# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../flite/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fixed compilation with gcc >= 3.4.

  - Rene Rebe <rene@exactcode.de>

--- flite-1.2-release/include/cst_sts.h.vanilla	2005-03-23 16:10:16.000000000 +0100
+++ flite-1.2-release/include/cst_sts.h	2005-03-23 16:10:23.000000000 +0100
@@ -47,9 +47,9 @@
 /* else where, this information plus the indexes in the Unit relation */
 /* allow reconstruction of the signal itself                          */
 struct cst_sts_struct {
-    const unsigned short *frame;  
-    const int size;      /* in samples */
-    const unsigned char *residual;
+    unsigned short *frame;  
+    int size;      /* in samples */
+    unsigned char *residual;
 };
 typedef struct cst_sts_struct cst_sts;
 
